Already in debcargo-conf.git, can be deleted for next source import.

--- a/vendor/curl-sys/build.rs
+++ b/vendor/curl-sys/build.rs
@@ -4,7 +4,6 @@
 use std::process::Command;
 
 fn main() {
-    println!("cargo:rerun-if-changed=curl");
     let host = env::var("HOST").unwrap();
     let target = env::var("TARGET").unwrap();
     let windows = target.contains("windows");
